src/amo/components/SearchForm/index.js

```diff
@@ -199,8 +222,8 @@ export class SearchFormBase extends React.Component {
           onSuggestionsClearRequested={this.handleSuggestionsClearRequested}
           onSuggestionsFetchRequested={this.handleSuggestionsFetchRequested}
           onSuggestionSelected={this.handleSuggestionSelected}
+          ref={(ref) => { this.autocomplete = ref; }}
```

Comment:
No biggie but I think this would make more sense as `this.autosuggest` since it's an instance of that component.

Reply:
Oh, totally. I'll change it, cheers!